app/components/root.jsx

```diff
@@ -44,7 +45,9 @@ export default class Root extends Component {
             <Route path="/myaccount" component={MyAccount} />
             <Route path="/checkout" component={CheckoutGuestAddress} exact />
             <Route path="/checkout2" component={CheckoutGuestPay} exact />
-            <Route path="/success" component={Success} exact />
+            <Route path="/success/:id" component={Success} exact />
+            <Route path="/addproduct" component={AddProduct} exact />
+            <Route path="/editprofile" component={EditUserProfile} exact />
```

Comment:
i want to see a 404 route at the end of the switch

Reply:
will be in once 'DRY Text Fields and 404 page' PR is accepted